c2corg_ui/static/js/viewdetails.js

```diff
@@ -168,6 +222,77 @@ app.ViewDetailsController.prototype.toggleTab = function(tab) {
 
 
 /**
+ * blablabla
+ * @export
+ */
+app.ViewDetailsController.prototype.initHeadband = function() {
+  if (this.documentService.document.activities.indexOf('rock_climbing') > -1 || this.documentService.document.activities.indexOf('mountain_climbing') > -1 || this.documentService.document.activities.indexOf('ice_climbing') > -1) {
+    this.hasHeadband = false;
+    if (this.documentService.document.associations.images.length == 0) {
+      this.hasVerticalImg = false;
+    } else {
+      this.hasVerticalImg = true;
+    }
+  } else {
+    this.hasVerticalImg = false;
+    if (this.documentService.document.associations.images.length == 0) {
+      this.hasHeadband = false;
+    } else if (this.documentService.document.associations.images.length == 1) {
+      this.hasHeadband = true;
+      this.scope_.headBands = this.createImageUrl(this.documentService.document.associations.images[this.widestImg_]['filename'],'BI');
+    } else if (this.documentService.document.associations.images.length > 1) {
+      this.getBestWideImg(0);
+      this.hasHeadband = true;
+    } else {
+      this.hasHeadband = false;
+    }
```

Comment:
This block is useless because at least one of the 3 previous conditions (length == 0 / length == 1 / length > 1) is always true.
```
this.hasHeadband = !!this.documentService.document.associations.images;
if (this.documentService.document.associations.images.length == 1) {
  this.scope_.headBands = this.createImageUrl(this.documentService.document.associations.images[this.widestImg_]['filename'],'BI');
} else if (this.documentService.document.associations.images.length > 1) {
  this.getBestWideImg(0);
}
```